lib/msf/core/exploit/local/persistence.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: binary -*-
+
+module Msf
+  module Exploit::Local::Persistence
+    def initialize(info = {})
+      super(
+        update_info(
+          info,
+          'DefaultOptions' => {
+            'WfsDelay' => 90_000, # 25hrs
+            'AllowNoCleanup' => true # don't delete our persistence after we get a shell
+          },
+          # https://github.com/rapid7/metasploit-framework/pull/19676#discussion_r1907594308
+          'Stance' => Msf::Exploit::Stance::Passive
+          # 'Passive' => true # XXX when set, ignores wfsdelay and immediately exists after last command
+        )
+      )
+
+      register_advanced_options(
+        [
+          OptString.new('WritableDir', [true, 'A directory where we can write files', '/tmp/'])
```

Comment:
Since this isn't tied to a specific platform, what if we left the default value blank then determined it at runtime and set it to something sensible based on the target platform.

Reply:
That's fine, is there a way to set the value in the module /wo overwriting all the text? Something like
`set_option('WritableDir', '/tmp/')` or do I have to overwrite it like:
`OptString.new('WritableDir', [true, 'A directory where we can write files', '/tmp/'])`